### DISCUSSION TITLE
How do I get sample project code

### DISCUSSION THREAD

**Discussion by Niceyaa** [pixijs/pixijs#9503] [Q&A] [answered]
Sorry, I want to get the example project code of 'example' on the official website. What channels or methods should I use to get it? Thank you very much.

**Comment by ProgrammingLife** [comment]
Just go to: https://pixijs.io/examples/#/demos-basic/container.js - copy that example code (or any other code you want), then paste it to your html file, include PIXI and run in the browser.

**Reply by ProgrammingLife** [reply]
But be careful with images - you need to have them if example uses some of them. It's also a good practice to start a local webserver in your folder (where your html file is placed). In easy way you can do it from the console:
```
python -m http.server
```
And then run your html in the browser: http://127.0.0.1/your.html

**Reply by Niceyaa** [reply]
thanks

**Comment by SuperSodaSea** [comment, marked as answer]
Old repo (for <https://pixijs.io/examples>): <https://github.com/pixijs/examples>
New repo (for <https://pixijs.com/examples>): <https://github.com/pixijs/pixijs.com/tree/main/src/data/examples>

**Reply by Niceyaa** [reply]
thanks
